Update npkill.js.org

zaldih opened the pull request:
Previously the website was in the project's own repo. Now I have decided to separate it and create it own repo for it.

- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (approved):
✅ Commit valid, CNAME present, site appears to have content \*

\* based off https://github.com/voidcosmos/npkill-web/blob/main/src/pages/index.astro

zaldih commented:
Woops! you're right. Apologies.

I've deleted and re-added the cname in the new repo. I also deleted ‘Custom domain’ in GitHub page settings in case it matters and completely disabled pages in the old repo.

I can still see the redirects, but from a vpn I see the 404 github page, so I guess it will be fixed in a while as soon as the dns propagate.

Thank you very much

indus commented:
I'm getting a 404 (from GitHub) now as well. I think something is still not configured correct in the repo. You can check availability on https://www.geopeeker.com/ (or some similar service)Am 25.04.2024 16:06 schrieb Juan Torres ***@***.***>:
Woops! you're right. Apologies.
I've deleted and re-added the cname in the new repo. I also deleted ‘Custom domain’ in GitHub page settings in case it matters and completely disabled pages in the old repo.
I can still see the redirects, but from a vpn I see the 404 github page, so I guess it will be fixed in a while as soon as the dns propagate.
Thank you very much

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

indus commented:
I think there is a misunderstanding. Before and after no kill.js.org points to voidcosmos.github.ioSo there is no change needed in JS.ORGs zonefile.The repo path in the list as well as the whole list itself is just for record keeping and having a change history. It has no functional purpose regarding DNS.I make changes to the zonefile first (not needed this time) and merge the PR only when the page is accessible and the CNAME redirect works.So you have to fix the GitHub side...Am 25.04.2024 18:01 schrieb Juan Torres ***@***.***>:
Maybe I'm missing something, but isn't that what you should get?
npkill.js.org currently points to voidcosmos.github.io/npkill
voidcosmos.github.io/npkill now no longer has any web content, because it's moved to voidcosmos.github.io/npkill-web,which is just the change I am requesting in this PR.
(The site does not display correctly because it expects to load resources from npkill.js.org when it is updated).

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

indus commented:
I think there is a misunderstanding.
Before and after `kill.js.org` actualy  points to `voidcosmos.github.io`.
So there is no change needed in JS.ORGs zonefile.

The repo path in the list as well as the whole list itself is just for record keeping and having a change history. It has no functional purpose regarding DNS.

I usually make changes to the zonefile first (not needed this time) and merge the PR only when the page is accessible and the CNAME redirect works.

So you have to fix the GitHub side...

zaldih commented:
Agreed, you were absolutely right.

I hadn't thought of that because I didn't quite know how github handled this underneath. That, plus for some reason github deployed with Actions was going into the `ERR_TOO_MANY_REDIRECTS` loop and so I was confused.
I moved the files to gh-page as some people recommended and now it works fine. However I will contact gh to see what's going on.

Thanks again and sorry for the problem.

indus commented:
Welcome back https://npkill.js.org
